### PR TITLE
Reorder meson status output

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -3746,92 +3746,101 @@ found = []
...
+        ['qrencode'],
+        ['tpm2'],
+        ['xkbcommon'],
+
+        # compression libs
+        ['zstd'],
+        ['lz4'],
+        ['xz'],
+        ['zlib'],
+        ['bzip2'],
```
Comment (poettering) [inline meson.build:3783]: maybe order alphabeticall within each section?

**Comment by poettering** [comment]
lgtm.

the tuples array might be something to put together in multiple independent assignments though? i.e. one for each "section"? i.e. "foo = […]; " and then "foo += […]" for each "section"?

File: meson.build (hunk 1 above)
Comment (keszybz) [reply meson.build:3783]: I ordered the compressors by "usefulness" (somewhat vaguely defined, but e.g. bzip2 deserves to be last...).

**Comment by keszybz** [comment]
> the tuples array might be something to put together in multiple independent assignments though? i.e. one for each "section"? i.e. "foo = […]; " and then "foo += […]" for each "section"?

That'd just be more boilerplate, and the comment would need to still be there. So I don't think this would gain anything.

**Pull request merged**